includes/Utilities/DomHelpers.php

```diff
@@ -125,4 +129,28 @@ public static function getTextFromSelector( $html, $selector ) {
 		// https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent
 		return $nodes[0]->text();
 	}
+
+	/**
+	 * Parses the html into DOMElement and searches the DOM tree for a given XPath expression or CSS selector.
+	 *
+	 * @param string      $html The HTML string to parse.
+	 * @param string|null $selector The selector to use.
+	 *
+	 * @return \DOMElement[]|\DOMElement
+	 */
+	public static function findNodes( $html, $selector = null ) {
+		$value = null;
```

Comment:
My IDE is complaining about `$value` always being `null`. Could rename it to `$elem` to align with what's happening below.